Description: Various autotools fixes.
 In particular:
 .
 Avoid recursive prefix (files were being installed in DESTDIR/DESTDIR/usr).
 .
 Avoid "will not overwrite just-created" error caused by installing logo.xpm
 twice.
 .
 Avoid "AC_RUN_IFELSE was called before AC_USE_SYSTEM_EXTENSIONS" and
 "AC_COMPILE_IFELSE was called before AC_USE_SYSTEM_EXTENSIONS" warnings.
 .
 Avoid " 'INCLUDES' is the old name for 'AM_CPPFLAGS' (or '*_CPPFLAGS')"
 warning.
 .
 Avoid "AC_CACHE_VAL(ac_have_backtrace, ...): suspicious cache-id, must
 contain _cv_ to be cached"
 .
 Avoid " The 'AM_PROG_MKDIR_P' macro is deprecated, and its use is
 discouraged."
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2014-10-18

--- a/Makefile.am
+++ b/Makefile.am
@@ -96,7 +96,7 @@
 MAINT_CHARSET = latin1
 
 # pour faire plaisir à woof :°)
-prefix=$(DESTDIR)@prefix@
+#prefix=$(DESTDIR)@prefix@
 
 SUBDIRS = intl  m4  po src scripts platypus
 EXTRA_DIST = config.rpath  MACOSX xpms/porte.h xpms/editwin_minib.xcf xpms/editwin_minib.xpm \
@@ -125,6 +125,6 @@
 	(cd ..; ./up.sh ./wmcoincoin-cvs.tar.gz download/devel)
 
 pkgdata_DATA = options README wmcoincoin.spec xpms/bracket.xpm  xpms/logo.xpm xpms/badnews.xpm \
-	xpms/totoz.xpm pixmaps/cbronson.jpg pixmaps/logoh.png pixmaps/logo.xpm xpms/icon.xpm
+	xpms/totoz.xpm pixmaps/cbronson.jpg pixmaps/logoh.png xpms/icon.xpm
 
 CLEANFILES = stamp-h
--- a/platypus/Makefile.am
+++ b/platypus/Makefile.am
@@ -5,5 +5,5 @@
 
 wmcoincoin_player_SOURCES = fmalloc.c giffunc.c gifread.c wmcoincoin_player.c gif.h
 
-INCLUDES = -I. `imlib2-config --cflags`
+AM_CPPFLAGS = -I. `imlib2-config --cflags`
 
--- a/configure.ac
+++ b/configure.ac
@@ -240,6 +240,8 @@
          * ) CYGWIN=no; EXEEXT="";;
 esac
 
+AC_USE_SYSTEM_EXTENSIONS
+
 dnl internationalization macros
 ALL_LINGUAS="fr"
 AM_GNU_GETTEXT
@@ -463,14 +465,14 @@
 	AC_DEFINE(HAVE_STRUCT_ADDRINFO,1,[struct addrinfo])
 fi
 
-AC_CACHE_CHECK([for execinfo.h and backtrace], ac_have_backtrace, [
+AC_CACHE_CHECK([for execinfo.h and backtrace], ac_cv_have_backtrace, [
         AC_TRY_COMPILE(
                 [ #include <execinfo.h>       
                 ],
                 [ void* trace[256]; int n = backtrace(trace, 256); ],
-                [ ac_have_backtrace="yes" ],
-                [ ac_have_backtrace="no"  ])])
-if test "x$ac_have_backtrace" = "xyes"; then
+                [ ac_cv_have_backtrace="yes" ],
+                [ ac_cv_have_backtrace="no"  ])])
+if test "x$ac_cv_have_backtrace" = "xyes"; then
         AC_DEFINE(HAVE_BACKTRACE,1,[glibc backtrace function])
 fi;     
 
--- a/m4/intl.m4
+++ b/m4/intl.m4
@@ -25,7 +25,7 @@
 AC_DEFUN([AM_INTL_SUBDIR],
 [
   AC_REQUIRE([AC_PROG_INSTALL])dnl
-  AC_REQUIRE([AM_PROG_MKDIR_P])dnl defined by automake
+  AC_REQUIRE([AC_PROG_MKDIR_P])dnl defined by automake
   AC_REQUIRE([AC_PROG_CC])dnl
   AC_REQUIRE([AC_CANONICAL_HOST])dnl
   AC_REQUIRE([gt_GLIBC2])dnl
--- a/m4/po.m4
+++ b/m4/po.m4
@@ -24,7 +24,7 @@
 [
   AC_REQUIRE([AC_PROG_MAKE_SET])dnl
   AC_REQUIRE([AC_PROG_INSTALL])dnl
-  AC_REQUIRE([AM_PROG_MKDIR_P])dnl defined by automake
+  AC_REQUIRE([AC_PROG_MKDIR_P])dnl defined by automake
   AC_REQUIRE([AM_NLS])dnl
 
   dnl Release version of the gettext macros. This is used to ensure that
